clients/src/main/java/org/apache/kafka/clients/consumer/internals/RequestState.java

```diff
@@ -132,6 +132,16 @@ public void onFailedAttempt(final long currentTimeMs) {
         this.numAttempts++;
     }
 
+    /**
+     * Set backoff and number of attempts to 0. This will ensure that the request is sent out
+     * again right away. Expected to be used when receiving errors responses that lead to a new
+     * request that can be sent without backing off (ex. member rejoining after fencing)
+     */
+    public void resetBackoff() {
+        this.backoffMs = 0;
```

Comment:
Should we use `exponentialBackoff.backoff(0)` as initial value? This seems to be what we do on successful requests and in `reset`.

Reply:
sure, got it from the existing reset now.